src/UTF-unknown.csproj

```diff
@@ -1,17 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard1.3;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard1.3;net40;netcoreapp3.0</TargetFrameworks>
   </PropertyGroup>
+
   <PropertyGroup>
     <AssemblyName>UtfUnknown</AssemblyName>
     <PackageId>UTF.Unknown</PackageId>
-    <Version>2.0.0</Version> <!-- patched in AppVeyor.yml -->
-
+    <Version>2.1.0</Version> <!-- patched in AppVeyor.yml -->

```

Comment:
FYI this value is ignored on the CI (appveyor), where the releases are created.

Reply:
Ok. Should I return the changes?

Reply:
Please do. Less changes = better :)

Reply:
Ok